.env.private

```diff
@@ -27,8 +27,8 @@ REACT_APP_UNCHAINED_BITCOINCASH_HTTP_URL=https://api.bitcoincash.shapeshift.com
 REACT_APP_UNCHAINED_BITCOINCASH_WS_URL=wss://api.bitcoincash.shapeshift.com
 REACT_APP_UNCHAINED_COSMOS_HTTP_URL=https://api.cosmos.shapeshift.com
 REACT_APP_UNCHAINED_COSMOS_WS_URL=wss://api.cosmos.shapeshift.com
-REACT_APP_UNCHAINED_THORCHAIN_HTTP_URL=https://api.thorchain.shapeshift.com
-REACT_APP_UNCHAINED_THORCHAIN_WS_URL=wss://api.thorchain.shapeshift.com
+REACT_APP_UNCHAINED_THORCHAIN_HTTP_URL=https://rpc.ninerealms.com
+REACT_APP_UNCHAINED_THORCHAIN_WS_URL=wss://rpc.ninerealms.com
```

Comment:
Something looks off here.

In `.env.app` we are doing:

```diff
- REACT_APP_UNCHAINED_THORCHAIN_HTTP_URL=https://api.thorchain.shapeshift.com
+ REACT_APP_UNCHAINED_THORCHAIN_HTTP_URL=https://api.ninerealms.com
```

but here in `.env.private` we are doing:

```diff
- REACT_APP_UNCHAINED_THORCHAIN_HTTP_URL=https://api.thorchain.shapeshift.com
+ REACT_APP_UNCHAINED_THORCHAIN_HTTP_URL=https://rpc.ninerealms.com
```